pkg/cloud/openstack/cluster/actuator.go

```diff
@@ -42,6 +42,10 @@ func NewActuator(params providerv1openstack.ActuatorParams) (*Actuator, error) {
 // Reconcile creates or applies updates to the cluster.
 func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
 	klog.Infof("Reconciling cluster %v.", cluster.Name)
+	if cluster == nil {
```

Comment:
This checking should right before the log at Line44?

Reply:
right, I didn't pay attention, thanks for pointing out